.github/workflows/flatpak.yml

```diff
@@ -7,6 +7,7 @@ on:
   pull_request:
     paths-ignore: ['**.md']
     branches: [master]
+    types: [ opened, synchronize, reopened, labeled ]
```

Comment:
Nitpick: please remove the space before and after the brackets - the rest of the files doesn't do that.

Another nitpick: please sort these alphabetically → `types: [labeled, opened, reopened, synchronize]`

Reply:
Sure